### DISCUSSION TITLE
please tell me

### DISCUSSION THREAD

**Discussion by kebab03** [JacquesLucke/animation_nodes#1871] [Q&A]
what is the equivalent of Object ->from group of AN2.0.4 in AN2.2.2 .Thank you in advance

**Comment by OmarEmaraDev** [comment]
You are probably looking for the Collection Info node.

**Reply by kebab03** [reply]
I was following  the tutorial from https://www.youtube.com/watch?v=jEYI4lnOprY&t=596s . can you tell what can be substitute  the block/ node from group  & how it can be done?Thank you in advance .

**Reply by OmarEmaraDev** [reply]
All of those nodes are still available in the current version, which node are you talking about exactly?
If you are asking about loop nodes, you should probably read this page first:
https://docs.animation-nodes.com/documentation/subprograms/loop/

**Reply by kebab03** [reply]
The node is called FROM GROUP.

**Reply by OmarEmaraDev** [reply]
As I mentioned, you are probably looking for the Collection Info node.
https://docs.animation-nodes.com/documentation/nodes/object/collection_info/
